spark/spark.cpp

```diff
@@ -474,7 +504,7 @@ namespace RayGene3D
     );
   }
 
-  void Spark::CreateSkyboxTriangles()
```

Comment:
I beleive skybox can be rendered in single pass with geometry (right after and w/o targets changes)